WooCommerce/Classes/ViewRelated/Products/Categories/ProductCategoryViewModelBuilder.swift

```diff
@@ -43,6 +43,15 @@ extension ProductCategoryListViewModel {
             }
         }
 
+        /// Returns a simple array of `ProductCategoryCellViewModel` from the provided `categories` disregarding the parent-child relationships.
+        /// Provide an array of `selectedCategories` to properly reflect the selected state in the returned view model array.
+        ///
+        static func flatViewModels(from categories: [ProductCategory], selectedCategories: [ProductCategory]) -> [ProductCategoryCellViewModel] {
+            return categories.map { category in
```

Comment:
What do you think about removing `return` here?

Reply:
Done in 1f5beb3da5107bb09a4901788653d34eb4f306f4.